drivers/afe/ad4110/ad4110.c

```diff
@@ -148,14 +148,91 @@ int32_t ad4110_set_adc_mode(struct ad4110_dev *dev, enum ad4110_adc_mode mode)
  * @return SUCCESS in case of success, negative error code otherwise.
 *******************************************************************************/
 int32_t ad4110_set_gain(struct ad4110_dev *dev, enum ad4110_gain gain)
+{
+	return ad4110_spi_int_reg_write_msk(dev,
```

Comment:
not sure actually if all these wrapper functions are needed. You can just call `ad4110_spi_int_reg_write_msk` inline.

Reply:
are these wrappers that call only one spi_write function requested explicitely by someone? otherwise i'd suggest to drop them.

Reply:
These wrappers are created for easier setup. If wanted, settings can be changed using wrappers even after ad4110_setup call.